### ISSUE TITLE
Connection String error on build all command

### ISSUE THREAD

**Issue by pancawardana** [Polyrific-Inc/OpenCatapult#412]
**Describe the bug**
I got connection string error once choose default connection string in build all process

**To Reproduce**
1. Open Power Shell
2. Run build all command

**Screenshots**
![Screenshot1](https://user-images.githubusercontent.com/48673584/54655624-351c4980-4af5-11e9-8500-07cf9e5425f3.png)

**Environment (please complete the following information):**
- Component: CLI
 - OS: Windows
 - Version  1.0.0 Beta2

**Comment by BrainBalaka** [comment]
@pancawardana I've just tried this on my machine and it gives me a different error:
![image](https://user-images.githubusercontent.com/5100736/55135969-abf1bc00-515f-11e9-849a-392c5df6a257.png)
Which is expected, because the sa password is not correct.

I notice in your screenshot there's a message saying `Connection string has been updated`. It seems like you've entered some white spaces when prompted for the connection string. I'll create a PR to modify the build script to handle this scenario
